web/src/app/administration/components/classification-details/classification-details.component.ts

```diff
@@ -80,7 +78,9 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
     this.action$.pipe(takeUntil(this.destroy$)).subscribe(data => {
       this.action = data;
       if (this.action === ACTION.CREATE) {
-        this.initClassificationOnCreation();
+        this.selectedClassification$.pipe(take(1)).subscribe(initialClassification => {
+          this.classification = { ...initialClassification };
```

Comment:
why are we copying the classification?

Reply:
Because the initialClassication from store is read only and we want to edit it while creating this new classification

Reply:
:+1: